defaults/main.yml

```diff
@@ -2,6 +2,12 @@ rhel7stig_cat1_patch: yes
 rhel7stig_cat2_patch: no
 rhel7stig_cat3_patch: no
 
+# These values match patch values by defaults. To override these values,
+# set them in group_vars, host_sars, or with the -e flag via CLI
+rhel7stig_cat1_audit: yes
```

Comment:
As a default, I would set the audit value to its corresponding patch value. That way they are "linked" by default but they can be controlled separately by the user.

```yaml
rhel7stig_cat1_audit: "{{ rhel7stig_cat1_patch }}"
rhel7stig_cat2_audit: "{{ rhel7stig_cat2_patch }}"
rhel7stig_cat3_audit: "{{ rhel7stig_cat3_patch }}"
```